tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.psd1

```diff
@@ -29,13 +29,14 @@ CompanyName = 'Microsoft Corporation'
 Copyright = 'Microsoft Corporation. All rights reserved.'
 
 # Description of the functionality provided by this module
-Description = 'Microsoft Azure PowerShell Predictor: Provide prediction while user types Azure PowerShell commands'
+Description = 'Microsoft Azure PowerShell - Module providing recommendations for Az cmdlets to PSRreadline 2.2 or above - This module is compatible with PowerShell 7.1.'
 
 # Minimum version of the PowerShell engine required by this module
 PowerShellVersion = '7.1'
 
 # Modules that must be imported into the global environment prior to importing this module
 # RequiredModules = @()
+RequiredModules = @(@{ModuleName="PSReadLine"; ModuleVersion="2.2.0-beta1"})
```

Comment:
I suggest to use ExternalModuleDependencies . Otherwise, the sign job requires PSReadLine  but it's not easy to integrate it in our build and sign job.

Here is one discussion.
https://powershell.org/forums/topic/externalmoduledependencies-requiredmodules/